nyc_taxis/TEST_PROCEDURES.md

```diff
@@ -16,13 +17,21 @@ The Searchable Snapshots feature is supported by OpenSearch since version 2.4.0,
 
 In addition to those mentioned in [README](README.md) of NYC taxis workload:
 * `snapshot_repository_name` (default: "test-repository"): Name of the snapshot repository.
-* `snapshot_name` (default: "test-snapshot"): Name of the snapshot. 
-  It is recommended to assign a different value for different test executions, because there is no operation defined by opensearch-benchmark to delete a snapshot, and a new snapshot will not be created if a previous snapshot with the same name exists in the repository.
+* `snapshot_name` (default: "test-snapshot"): Name of the snapshot.
 * `s3_bucket_name`: Name of the Amazon S3 bucket that stores the snapshot. The S3 bucket needs to be prepared manually.
 * `s3_bucket_region`: The AWS Region where the Amazon S3 bucket exists. For example, "us-east-1".
 
+Example:
+```
+{
+  "s3_bucket_name": "name of your S3 bucket",
+  "s3_bucket_region": "region of your S3 bucket"
+}
+ ```
+Save it as `params.json` and provide it to Benchmark with `--workload-params="/path/to/params.json"`.
+
 #### The test procedure requires parameters to be provided for the `repository-s3` plugin using `--plugin-params`:
-See the [Benchmark docs](https://github.com/opensearch-project/opensearch-benchmark/blob/main/osbenchmark/resources/provision_configs/main/plugins/v1/repository_s3/README.md
+See the [Benchmark docs](https://github.com/opensearch-project/opensearch-benchmark/blob/0.2.0/osbenchmark/resources/provision_configs/main/plugins/v1/repository_s3/README.md
```

Comment:
Was this changed because main might not always have this exact README.md in the future?

Reply:
Yes, that was my reason. What's your opinion? I can change back to use main branch.

Reply:
I'm fine with either! Just wanted to understand the rationale behind it

Reply:
@gkamat I agree. Since the whole file "TEST_PROCEDURES.md" is added by me, making changes can hardly impact the original "nyc_taxis" workload. I can move the test procedure "searchable snapshots" into a dedicated workload when needed.